.github/ISSUE_TEMPLATE/feature-regression.yml

```
@@ -0,0 +1,105 @@
+name: "\U0001F4A5 Feature Regression"
+description: Report a feature regression issue with `rolldown-vite`
+labels: ["c: regression", pending triage]
+type: Bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a feature regression. A regression is when a feature that works as expected when using `vite` no longer behaves correctly.
+  - type: textarea
+    id: regression-description
+    attributes:
+      label: Describe the regression
+      description: A clear and concise description of the regression. If you intend to submit a PR for this issue, let us know in the description. Thanks!
+      placeholder: The feature used to work like ... Now it behaves like ...
+    validations:
+      required: true
+  - type: input
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a link via [StackBlitz](https://stackblitz.com/fork/github/vitejs/rolldown-vite/tree/rolldown-vite/packages/create-vite/template-vanilla) or a link to a repo that can reproduce the regression issue. A [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) is required ([Why?](https://antfu.me/posts/why-reproductions-are-required)). If no reproduction is provided after 3 days, the issue may be auto-closed.
+      placeholder: Reproduction URL
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: Describe what you expected to happen when using the feature.
+      placeholder: I expected the feature to ...
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual Behavior
+      description: Describe what actually happens when using the feature.
+      placeholder: The feature currently behaves like ...
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide any reproduction steps that may need to be described. E.g., if it happens only when running the dev or build script, make sure it's clear which one to use.
+      placeholder: Run `npm install` followed by `npm run dev`
+  - type: textarea
+    id: system-info
+    attributes:
+      label: System Info
+      description: Output of `npx envinfo --system --npmPackages '{vite,@vitejs/*,rollup}' --binaries --browsers`
```

```suggestion
      description: Output of `npx envinfo --system --npmPackages '{vite,rolldown-vite,@vitejs/*,rollup,rolldown}' --binaries --browsers`
```
I didn't try, but maybe we should add `rolldown-vite` and `rolldown` here.

---

`rolldown` is a good idea for sure! I didn't add `rolldown-vite` as it should be aliased most of the time - but I guess it doesn't hurt (will be omitted if not available anyway).